### PR TITLE
Feature/sniff rta makedata

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

- refresh RTA makedata to a stage that adds logging to foxx crud
- send error messages via arangosh logfile to testfailurest.txt
- enable sniffing

- [ ] :hankey: Bugfix
- [x] :pizza: New feature

### CODE REVIEW

File: js/client/modules/@arangodb/testsuites/rta_makedata.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ function makeDataWrapper (options) {
         count += 1;
         let args = pu.makeArgs.arangosh(this.options);
         args['server.endpoint'] = this.instanceManager.findEndpoint();
+        args['log.file'] = fs.join(fs.getTempPath(), `rta_out_${count}.log`);
```
Comment (jsteemann) [inline js/client/modules/@arangodb/testsuites/rta_makedata.js:112]: does this (new) logfile need to be garbage-collected at test end?

Comment (dothebart) [reply js/client/modules/@arangodb/testsuites/rta_makedata.js:112]: its created in the test-directory of this testsuite, and hence will (not) be collected along with the instance logs on (non-)success.

Comment (dothebart) [reply js/client/modules/@arangodb/testsuites/rta_makedata.js:112]: fixed.

**Review by naushniki — APPROVED** [review]
LGTM

**Pull request merged**